test/library/gui/widgets.Label.lua

```diff
@@ -0,0 +1,117 @@
+-- test -dhack/scripts/devel/tests -twidgets.Label
+
+--reload('gui')
+--reload('gui.widgets')
+
+local gui = require('gui')
+local widgets = require('gui.widgets')
+
+local xtest = {} -- use to temporarily disable tests (change `function xtest.somename` to `function xxtest.somename`)
+local wait = function(n)
+    delay(n or 30) -- enable for debugging the tests
+end
+
+function test.Label_correct_frame_body_with_scroll_icons()
+    local fs = defclass(fs, gui.FramedScreen)
+    fs.ATTRS = {
+        frame_style = gui.GREY_LINE_FRAME,
+        frame_title = 'TestFramedScreen',
+        frame_width = 10,
+        frame_height = 10,
+        frame_inset = 0,
+        focus_path = 'test-framed-screen',
+    }
```

Comment:
You can declare this class outside of the function so you can reuse it across tests.